tests/t151_recv_runcmd.py

```diff
@@ -42,11 +46,16 @@ def prerun(self, timeout):
         record_cmd += ['t-' + self.name]
         self.pr_debug('prerun command: ' + ' '.join(record_cmd))
         sp.call(record_cmd, stderr=sp.PIPE)
+
+        epolls.poll(timeout=timeout)
+
         self.recv_p.terminate()
 
         out = self.recv_p.communicate()[0].decode(errors='ignore')
         self.file_p.write(out)
+        self.file_p.flush()
         self.file_p.close()
```

Comment:
I think `close()` implies `flush()` but maybe it's better to have it explicitly.